test/NuGet.Services.Staging.Test.EndToEnd/StagingClient/IStagingClient.cs

```diff
@@ -23,7 +23,9 @@ interface IStagingClient
 
         Task<JObject> Index(string stageId);
 
-        Task<JObject> Query(string stageId, string queryString);
+        Task<JObject> Query(string stageId, string query, bool includePrerelease, int skip, int take);
+
+        Task<JObject> Autocomplete(string stageId, string q, string id, bool includePrerelease, int skip, int take);

```

Comment:
I'm assuming the `id` parameter here refers to the `packageId`. Perhaps rename it?